scripts/check_files_compliance.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
This script is run on the host itself (i.e. not in a docker). It seems ubuntu-18.04 uses Python 2 and ubuntu-22.04 uses Python 3, so we need to do some minor adaptations to the script.

NTC (informative)